.golangci.yml

```diff
@@ -48,6 +48,7 @@ linters-settings:
     statements: 50
 
 linters:
+  # please, do not use `enable-all`: it's deprecated and will be removed soon.
```

Comment:
I've been using disable-all from the start. This means that from now it's implicit?

Even if we have versioned config it's great in order to avoid unexpected things.

:clap:

Reply:
I did no backward-incompatible changes: by default some linters are enabled